implementation/domain/logic/jbb-board-impl/src/main/java/org/jbb/board/impl/forum/dao/ForumCategoryRepository.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2017 the original author or authors.
+ *
+ * This file is part of jBB Application Project.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.board.impl.forum.dao;
+
+import org.jbb.board.impl.forum.model.ForumCategoryEntity;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
```

Comment:
Admin can manage forum category order with API yet though `ForumCategoryService.moveCategoryToPosition(...)`. But `addCategory` method always adds category at the end, and then admin can change its position with buttons "move up"/"move down" in UI